lib/bin/supllib/readme.txt

```diff
@@ -0,0 +1,18 @@
+This is the SUPL Client library component that adds SUPL support to NCS. The
```

Comment:
Shouldn't this be in `.rst` format, not `.txt`?

Reply:
Yes